wiki/Tournaments/RDOC/2014/en.md

```diff
@@ -0,0 +1,305 @@
+---
+tags:
+  - RDOC
+  - RDOC2014
+---
+
+# Royal Danish osu!Cup 2014
+
+![RDOC logo](img/logo.png)
+
+The **Royal Danish osu!Cup 2014** (***RDOC2014***) was a national double-elimination 31-player 1v1 osu!standard tournament hosted by ![][flag_DK] [Mayoi Hachikuji](https://osu.ppy.sh/users/2114273). This was the first installment of the Royal Danish osu!Cup.
```

Comment:
```suggestion
The **Royal Danish osu!Cup 2014** (***RDOC 2014***) was a Danish 1v1 double-elimination osu!standard tournament hosted by ![][flag_DK] [Mayoi Hachikuji](https://osu.ppy.sh/users/2114273). It was the first installment of the Royal Danish osu!Cup.
```

Reply:
I dont really wanna call it Danish here since its made for participants from the full kingdom which includes multiple nationalities. eg Greenland, Faroe Islands

Reply:
o well, thinking about it again. It still makes sense. applied